train.py

```diff
@@ -29,11 +29,64 @@ def prepare_eval_data(dataset_dict, tokenizer):
     # Since one example might give us several features if it has a long context, we need a map from a feature to
     # its corresponding example. This key gives us just that.
     sample_mapping = tokenized_examples.pop("overflow_to_sample_mapping")
+    offset_mapping = tokenized_examples["offset_mapping"]
 
     # For evaluation, we will need to convert our predictions to substrings of the context, so we keep the
     # corresponding example_id and we will store the offset mappings.
+    tokenized_examples["start_positions"] = []
+    tokenized_examples["end_positions"] = []
     tokenized_examples["id"] = []
     #tokenized_examples['topic_id'] = []
+    inaccurate = 0
+    for i, offsets in enumerate(tqdm(offset_mapping)):
```

Comment:
this is fine for now but we should probably refactor the code that does this to avoid having a copy/paste in our codebase